docs/root/intro/version_history.rst

```diff
@@ -3,6 +3,7 @@ Version history
 
 1.9.0 (pending)
 ===============
+* access log: added a :ref:`JSON logging mode <config_access_log_format_dictionaries>` to output access logs in JSON format
```

Comment:
Very OCD nit: can you add a `.` to the end of the sentence?

Reply:
Done!